src/deviceOauthService.ts

```diff
@@ -11,7 +11,7 @@ import Transport from 'jsforce/lib/transport';
 import { AsyncCreatable, Duration, parseJsonMap, sleep } from '@salesforce/kit';
 import { HttpRequest, OAuth2Config } from 'jsforce';
 import { ensureString, isString, JsonMap, Nullable } from '@salesforce/ts-types';
-import * as FormData from 'form-data';
+import FormData from 'form-data';
```

Comment:
All changes like this are to make the import statement compatible with esbuild without hurting the existing compiling process of typescript compiler.